static/js/services/live-list.js

```diff
@@ -82,6 +83,23 @@ angular.module('inboxServices').factory('LiveListConfig',
                 scope.overdue = contact.lastVisitedDate <= oneMonthAgo;
                 scope.summary = $translate.instant('contact.last.visited.date', { date: relativeDayFilter(contact.lastVisitedDate, true) });
               }
+
+              scope.visits = {
+                count: $translate.instant('contacts.visits.count', { count: maxIntegerFilter(contact.visitCount, 99) }),
+                summary: contact.visitCount === 1 ?
+                  $translate.instant('contacts.visits.count.visit') :
+                  $translate.instant('contacts.visits.count.visits'),
```

Comment:
I haven't looked at what capabilities angular translate has, but a lot of them support plurals in the actual translation itself. This is useful as certain languages do plurals differently (I have no idea about the ones we support, but there are languages with no plurals, or 3-4 different states, etc)

Reply:
This is correct! Thanks for pointing it out!